### PR TITLE
update the "using EDB Cloud API" doc with latest account management behavior change

### PR DESCRIPTION
**Pull request by xugy99**
## What Changed?

This PR 

- added new content in the `product_docs/docs/edbcloud/beta/reference/index.mdx` to describe one new step required the end-user to follow when the user is trying to get a token for accessing the EDB Cloud API.

Please note the relevant changes to the EDB Cloud production system have already been released. 

## Checklist

Please check all boxes that apply (`[ ]` is unchecked, `[x]` is checked)

**Content**

- [ ] This PR adds new content
- [x] This PR changes existing content
- [ ] This PR removes existing content
- [ ] This PR is for a release, please add this tag:

### CODE REVIEW

**Review by skrivov-edb — APPROVED** [review]

File: product_docs/docs/edbcloud/beta/reference/index.mdx [hunk 1]
```diff
@@ -134,20 +135,56 @@ For example:
...
+- `token` - The bearer token used to access the EDB Cloud API.
+
+For example:
+```
+{
+  "token": "eyJhbGc.......0HFkr_19Vr7w"
+}
+```
+
+This token is similar to the previous access token but it is the one that EDB Cloud API recognizes.
```
Comment (drothery-edb) [inline product_docs/docs/edbcloud/beta/reference/index.mdx:181]:
```suggestion
This token, as opposed to the raw access token, is recognized by the EDB Cloud API.
```

File: product_docs/docs/edbcloud/beta/reference/index.mdx [hunk 2]
```diff
@@ -218,6 +255,15 @@ The response of this API call includes the `access_token`. For example:
...
 ```
 
+Store the raw access token and refresh token in environment variables for future use. For example:
+
+```
+RAW_ACCESS_TOKEN="eyJhbGc.......1Qtkaw2fyho"
+REFRESH_TOKEN="v1.MTvuZpu.......sbiionEhtTw"
+```
+
+The token got from this step is still the raw access token, you still need to follow the step of [Exchange for EDB Cloud token](#exchange-the-edbcloud-token-using-curl) for exchanging an EDB Cloud token.
```
Comment (drothery-edb) [inline product_docs/docs/edbcloud/beta/reference/index.mdx:265]:
```suggestion
The token you obtain from this step is the raw access token, you need to exchange this token for an EDB Cloud token. See [Exchange for EDB Cloud token](#exchange-the-edbcloud-token-using-curl) for more information.
```

File: product_docs/docs/edbcloud/beta/reference/index.mdx [hunk 3]
```diff
@@ -134,20 +135,56 @@ For example:
...
 - `slow_down` - slow down and use the suggested interval retrieved when you [requested the device code](#request-the-device-code-using-curl). To avoid receiving this error due to network latency, you should start counting each interval after receipt of the last polling request's response.
 - `expired_token` - you have not authorized the device quickly enough, so the `device_code` has expired. Your application should notify you that it has expired and to restore it.
 - `access_denied`
 
+## Exchange the EDB Cloud token using `curl`
+!!!note
+   The `get-token` script executes this step. You don't need to make this call if you are using the script.
+!!!
+
+Get the EDB Cloud token with the raw token got previously:
```
[the remark this answers is not in the document]
Comment (drothery-edb) [reply product_docs/docs/edbcloud/beta/reference/index.mdx:162]: @xugy99 , did you see this suggestion?

[the remark this answers is not in the document]
Comment (xugy99) [reply product_docs/docs/edbcloud/beta/reference/index.mdx]: @drothery-edb yeah I saw it last night, it was quite late (I am located in Beijing) and there 's no button for me to click as the automatically merge so I just address this in the latest commit manually 😄

**Comment by xugy99** [comment]
All the suggestion are addressed in the latest commits. @drothery-edb please review that in another round.

**Pull request merged**